.github/workflows/bloat_check.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
     pull_request_update:
         name: Report on pull requests
 
-        runs-on: ubuntu-latest
```

Comment:
This seems ok, but does it work? The line 33 below uses chip-build which is still `FROM ubunut:focal` so I am guessing we just change host not the underlying OS.

Can we somehow validate if this is what we want or if we instead want to change our chip-build and underlying docker images?

Reply:
100%. I'll update images too. Let me update those as well